### ISSUE TITLE
Attribute has already been defined

### ISSUE THREAD

**Issue by RobbySmet** [nohana/Laevatein#24]
Hi, when I try to use this library I get the following error while building:

C:\Users\Robby\Git repositories\Immozo\app\build\intermediates\exploded-aar\com.mcxiaoke.viewpagerindicator\library\2.4.1\res\values\values.xml

Error:Attribute "text" has already been defined
Error:Attribute "textColor" has already been defined
Error:Attribute "textSize" has already been defined

**Comment by elf-zwolf** [comment]
+1

**Comment by KeithYokoma** [comment]
Thanks for your feedback.

The problem is that Android build system doesn't take care about attribute names that are conflicting with other libraries, and there's no way to tell the system to avoid this situation for now.

It's quite messy but the only way to fix this problem that we need to rename our attribute names adding some prefix looks like "lvt_".

@takaya1219 Please review this solution.

**Comment by RobbySmet** [comment]
@KeithYokoma Yeah, I've seen the prefix solution used by numerous libraries so I think it's definitely a good solution.
